wonkyung/app.js

```diff
@@ -0,0 +1,45 @@
+require("dotenv").config();
+
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+
+
+
+
+const { DataSource } = require('typeorm')
+
+const appDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+})
+
+appDataSource.initialize()
+  .then(() => {
+      console.log("Data Source has been initialized!")
+  });
```

Comment:
데이터베이스 Connection에 실패했을 경우 에러를 반환하도록 수정해주세요.

```suggestion
appDataSource.initialize()
  .then(() => {
      console.log("Data Source has been initialized!")
  })
  .catch(() => {
      console.log("Failed to connect to Database")
  });
```